### ISSUE TITLE
Would it be alright to store the reverse key in addition to the regular key in an exchange?

### ISSUE THREAD

**Issue by RestitutorOrbis** [varunsrin/rusty_money#51]
If you store a rate for `USD  -> GBP` and then you query for the rate for `GBP -> USD`, you get a `None` value. This doesn't seem correct to me. Shouldn't the value returned be 1/(rate returned for `USD -> GBP`)?

**Comment by varunsrin** [comment]
yes, the exchange rate crate isn't very mature right now, thanks for pointing out these issues. 

i noticed some PR's, will review them over the weekend.

**Comment by RestitutorOrbis** [comment]
That sounds good, no rush 😄 . I actually probably should add some tests anyways.

**Comment by DavidBM** [comment]
We found that the rate for `USD -> GBP` is not the same as the rate for `GBP -> USD` in the real world exchanges (example: currencylayer.com). 

If the crate wants to align with the currency exchanges in the market, it should return `None` as in the majority of cases you will want to store two different rates.
